src/index.js

```diff
@@ -1,4 +1,9 @@
-import { factory, defaultConfig } from './Block';
-import { getDataProps } from './Block/data';
+import { createFactory } from '@volusion/element-block-utils';
+import { StarterBlockFactory } from './block';
+import { styles } from './styles';
+import { configSchema, defaultConfig } from './configs';
+import { getDataProps } from './data';
+
+const factory = createFactory(StarterBlockFactory, styles, configSchema);
```

Comment:
Are we going to update the fn params to use an object syntax?

Reply:
Yes, I think that's a good idea.

Reply:
Thinking about this a little more:

Using an object for the arguments:
`createFactory({ StarterBlockFactory, getStyles, configSchema })`
This style makes it more difficult to change the argument names (i.e. `styles()` -> `getStyles()`).
However, it makes it easier to remove arguments we no longer need (i.e. if we could start exporting `configSchema()` directly from index.js
It also makes it easier for users of the function to leave out arguments they don't need, for example if they have a `configSchema`, but no block styles.

Just having multiple arguments:
`createFactory(StarterBlockFactory, getStyles, configSchema)`
This style makes it easy to change the names of the arguments, but more difficult to remove them or change the order.

Alternately, we could take a combined approach, and make blockFactory one argument, with everything else in an config object:
`createFactory(StarterBlockFactory, { getStyles, configSchema })`

Reply:
@cameron-jones-volusion  hm... I think I like option 3 the most since everything in the object is potentially optional.

Reply:
I'll open a new PR for this once the `element-block-utils` package has been updated with this change.